build.cake

```diff
@@ -84,6 +83,7 @@ Setup(context =>
             Verbose("GitVersion:\n{0}", gitVersionInfo.Dump());
             Information("Building {1} images v{0}", gitVersionInfo.SemVer, dockerNamespace);
             dockerTag = new OctopusDockerTag(gitVersionInfo, dockerNamespace, imageDirectory);
+            Information($"##teamcity[setParameter name='WorkerToolsVersion' value='{dockerTag.version}']");
```

Comment:
It feels really weird to me to set a parameter inside `Information()` ... 🤔 

Are we able to use [the TeamcityProvider](https://cakebuild.net/api/Cake.Common.Build.TeamCity/TeamCityProvider/7F887BF0) to set this parameter instead?

Reply:
Updated 🙂